src/main/webapp/js/feedbackResponseComments.js

```diff
@@ -397,7 +391,6 @@ function loadFeedbackResponseComments(user, courseId, fsName, fsIndx, sender) {
             registerResponseCommentsEvent();
             registerCheckboxEventForVisibilityOptions();
             enableHoverToDisplayEditOptions();
-            enableTooltip();
```

Comment:
Yeah actually it was to fix something I missed in #3858 but I see what you mean, not good to hijack this PR, I will move it into another PR and issue (open a new one to indicate I missed this one and check if I missed anything else).